subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/app/CppApp.groovy

```diff
@@ -22,16 +22,17 @@ package org.gradle.nativeplatform.fixtures.app
 class CppApp extends CppSourceElement implements AppElement {
     final greeter = new CppGreeter()
     final sum = new CppSum()
+    final multiply = new CppMultiply()
```

Comment:
I don't think we should keep adding more stuff to this fixture (and the library and Swift equivalents) as this becomes very costly to maintain as more and more tests jam their specific needs into a single fixture. Instead, we should add fixtures for the 'add or remove source files' scenarios, or reuse them in the case of Swift tests (see `IncrementalSwiftApp` and `IncrementalSwiftLibrary`.